src/github.com/getlantern/fronted/dialer.go

```diff
@@ -183,14 +183,15 @@ func (d *dialer) Close() error {
 }
 
 func (d *dialer) HttpClientUsing(masquerade *Masquerade) *http.Client {
-	enproxyConfig := d.enproxyConfigWith(func(addr string) (net.Conn, error) {
-		return d.dialServerWith(masquerade)
-	})
-
 	return &http.Client{
-		Transport: &http.Transport{
-			Dial: func(network, addr string) (net.Conn, error) {
-				return enproxy.Dial(addr, enproxyConfig)
+		Transport: &DirectDomainTransport{
+			Transport: http.Transport{
```

Comment:
[not important] I stay away from the lantern repo code for a while and I forget about the relationship between clients, dialers, transports etc.- transports have transports inside?  :S